py/desispec/sky.py

```diff
@@ -177,6 +177,13 @@ def compute_uniform_sky(frame, nsig_clipping=4.,max_iterations=100,model_ivar=Fa
     skyfibers = np.where(frame.fibermap['OBJTYPE'] == 'SKY')[0]
     assert np.max(skyfibers) < 500  #- indices, not fiber numbers
 
+    #- Hack: test tile 81097 (observed 20210430/00086750) had set
+    #- FIBERSTATUS bit UNASSIGNED for sky targets on stuck positioners.
+    #- Undo that.
+    if (frame.meta is not None) and ('TILEID' in frame.meta) and (frame.meta['TILEID'] == 81097):
```

Comment:
Do we want to have special code for one test sv2 tile? The answer might well be yes, but we could also just flag the exposure as bad. I'm okay either way on this one

Reply:
I don't want to get in the habit of hardcoding special cases, but at the time I was writing this code this was the only tile that had skies assigned to stuck positioners and so I needed something like this for testing.  It seemed harmless enough that I left it in.